self-paced-labs/vertex-ai/vertex-ai-qwikstart/lab_exercise.ipynb

```diff
@@ -2,60 +2,72 @@
  "cells": [
```

Comment:
<p>Just realized you can actually drop the PROJECT ID from and just do</p><pre class="ql-syntax" spellcheck="false">FROM
  online_retail.online_retail_clv_ml
</pre><p>This way the person working on the notebook can actually just run the queries without replacing each FROM statement</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/training-data-analyst/pull/1570/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='26'/>

Reply:
Cash $$$! Thank you! Fixed.